### ISSUE TITLE
Wiked pdf does not show image alternate text

### ISSUE THREAD

**Issue by hmtanbir** [mileszs/wicked_pdf#847]
## Issue description
Hi, 
I have integrated wicked pdf with my spree project, when I add image with it's alt text, the alternate text is not showing while the image is missing in pdf.

here is my code,

`= link_to(wicked_pdf_image_tag(item.image_url, alt: 'image archived'), variants_link(item))`

## Expected or desired behaviour
Alternate text of image should be shown in pdf file.
## System specifications
Linux Mint 19.02

ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-linux]

rails 5.2.1

spree 3.6.3

wicked_pdf 1.1.0

Could you please prioritize this issue @unixmonkey?

**Comment by unixmonkey** [comment]
@hmtanbir I don't think I can. That is behavior from `wkhtmltopdf`, which this project is just a wrapper for.

You *might* be able to display alternate text yourself with CSS or JS by detecting when an image isn't loaded in a particular element.
